pkg/cidata/cidata.TEMPLATE.d/boot/40-install-containerd.sh

```diff
@@ -5,6 +5,28 @@ if [ "${LIMA_CIDATA_CONTAINERD_SYSTEM}" != 1 ] && [ "${LIMA_CIDATA_CONTAINERD_US
 	exit 0
 fi
 
+if [ "${LIMA_CIDATA_CONTAINERD_SYSTEM}" = 1 ] && [ -f /etc/alpine-release ]; then
+	# need to use the containerd packages
+	apk add runc containerd cni-plugins
+	rc-update add containerd default
+	rc-service containerd start
+
+	tar -xz -C /usr/local -f "${LIMA_CIDATA_MNT}"/nerdctl-full.tgz bin/buildctl bin/buildkitd
+	nohup buildkitd --oci-worker=false --containerd-worker=true >/var/log/buildkitd.log 2>&1 &
+
+	# normally set up by systemd-logind
+	cat >"/etc/profile.d/xdg_runtime.sh" <<EOF
+	if ! [ -d /tmp/runtime-\$USER ]; then
+	  mkdir -p /tmp/runtime-\$USER
+	  chmod 700 /tmp/runtime-\$USER
+	fi
+	export XDG_RUNTIME_DIR=/tmp/runtime-\$USER
```

Comment:
Why not use /run/user/$UID

Reply:
I wasn't really looking to reimplement loginctl and all the tmpfs and things, but maybe could just move it there.

Reply:
systemd compatibility seems to only be needed for rootless mode, so I will remove this shell script instead.